CONTRIBUTING.md

```diff
@@ -39,12 +39,25 @@ To sync your fork with the OpenMined/PySyft repository please see this [Guide](h
 
 ### Installing PySyft after Cloning Repository
 
-To install the development version of the package, once the `dev` version of the requirements have been satisified, one should follow the instructions as laid out in [INSTALLATION.md](https://github.com/OpenMined/PySyft/blob/master/INSTALLATION.md) to complete the installation process. Effectively do the following two steps after a clone has been made on one's local machine at the terminal and that the pre-commit hook has been set up as described above in [Setting up Pre-Commit Hook](#syncing-a-forked-repository):
-```bash
-cd PySyft
-pip install -e .
-```
-If you are using a virtual environment, please be sure to use the correct executable for `pip` or `python` instead.
+To install the development version of the package, once the `dev` version of the requirements have been satisified, one should:
+
+1. Follow the instructions as laid out in [INSTALLATION.md](https://github.com/OpenMined/PySyft/blob/master/INSTALLATION.md) to complete the installation process.
+2. Make a clone of PySyft repo on one's local machine at the terminal
+3. Set up the pre-commit hook as described above in [Setting up Pre-Commit Hook](#Setting-up-Pre-Commit-Hook)
+4. Do the following two steps:
+
+    ```bash
+    cd PySyft
+    pip install -e .
+    ```
+
+5. Install the PySyft-Proto dependency, thus:
```

Comment:
Q: Wouldn't be simpler to simply add the ```pysyft-proto``` dependency in the ```requirements_udacity.txt``` file?

Reply:
I see your point. I think it will be easier, however in this context that would be `requirements_dev.txt` instead. Then I would have to remove the last step. What do you think?

Reply:
requirements_dev.txt or both

Reply:
Note that I've changed the name from pysyft_proto to syft_proto and added it to PyPI